hudi-client/hudi-client-common/src/main/java/org/apache/hudi/io/HoodieAppendHandle.java

```diff
@@ -654,22 +640,6 @@ private HoodieLogBlock.HoodieLogBlockType pickLogDataBlockFormat() {
     }
   }
 
-  private static Map<HeaderMetadataType, String> getUpdatedHeader(Map<HeaderMetadataType, String> header, int blockSequenceNumber, long attemptNumber,
-                                                                  HoodieWriteConfig config, boolean addBlockIdentifier) {
-    Map<HeaderMetadataType, String> updatedHeader = new HashMap<>(header);
-    if (addBlockIdentifier && !HoodieTableMetadata.isMetadataTable(config.getBasePath())) { // add block sequence numbers only for data table.
-      updatedHeader.put(HeaderMetadataType.BLOCK_IDENTIFIER, attemptNumber + "," + blockSequenceNumber);
-    }
-    if (config.shouldWritePartialUpdates()) {
```

Comment:
@nsivabalan we need to keep this part with the partial update flag

Reply:
@jonvex Good catch! Could you fix it?

Reply:
I fixed it.